libs/designer-ui/src/lib/editor/base/utils/helper.ts

```diff
@@ -197,7 +197,8 @@ export const getDropdownOptionsFromOptions = (editorOptions: any): ComboboxItem[
 
   // handle cases where the displayName is not a string
   dropdownOptions = dropdownOptions.map((option) => {
-    const stringifiedDisplayName = typeof option.displayName === 'string' ? option.displayName : JSON.stringify(option.displayName);
+    const displayName = option.displayName ?? option.key ?? option;
+    const stringifiedDisplayName = typeof displayName === 'string' ? displayName : JSON.stringify(displayName);
```

Comment:
I think you were supposed to do this for the value field not displayName

Reply:
I've seen display names come in non-string types, there was a time where it was an object, another time it was a number. Both caused the designer to crash. This is just code to insure at least we don't crash